src/osmf/lib/HDSPlayer.as

```diff
@@ -54,6 +54,7 @@ package
     private var _playQueue:Boolean = false;
     private var _seekToEnd:Boolean = false;
     private var _initialPlay:Boolean = true;
+    private var _isReinitialize:Boolean = false;
```

Comment:
Not sure what this variable is tracking, is there a more descriptive name that can be used? It's not clear why this is set in a closed captions callback or why it results in a play being queued.